src/main/docker/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM centos:7.5.1804
+
+LABEL maintainer="Eiffel-Community"
+
+ARG URL
+
+RUN yum -y install openssh-client initscripts sudo
+
+# Add Gerrit packages repository
+RUN rpm -i https://gerritforge.com/gerritforge-repo-1-2.noarch.rpm
+
+# Install OpenJDK and Gerrit in two subsequent transactions
+# (pre-trans Gerrit script needs to have access to the Java command)
+RUN yum -y install java-1.8.0-openjdk
+RUN yum -y install gerrit-2.14.16-1
+RUN yum -y install bash
+
+USER gerrit
+RUN java -jar /var/gerrit/bin/gerrit.war init --batch -d /var/gerrit
+RUN java -jar /var/gerrit/bin/gerrit.war reindex -d /var/gerrit
+
+COPY ${URL} /var/gerrit/plugins/eiffel-plugin.jar
```

Comment:
If you change to ADD, then docker image can be built based on a url address to plugin artifact and not just a filesystem path to artifact.

Reply:
Will change back, we must update codacy not to complain about that.

Reply:
I am not that good on docker but according to https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy is sounds like the "Use Copy instead" rule still applies. I think their idea of not using implicit gets is a good idea. What is your view here?

Reply:
We use ADD for building Docker image based on both local built application artifact and released applicaitons artifacts from artifactory, e.g. Jitpack or similar.
In some components we use the auto extract tar file functionality that comes with ADD function/method.

Reply:
As the best practice says *Because image size matters, using ADD to fetch packages from remote URLs is strongly discouraged* we should use COPY...